rules/iam_rules.yaml

```diff
@@ -41,6 +41,19 @@ rules:
         members:
           - allUsers
 
+  - name: Allow only members in my domain to be an OrgAdmin
```

Comment:
Please update this name to reflect what this rule does.

Reply:
Done